package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "@babel/runtime": "7.10.1",
     "bluebird": "2.9.25",
-    "forest-express": "7.8.1",
+    "forest-express": "7.9.1",
```

Comment:
No update for the `yarn.lock` file?

Reply:
I will have too, once `forest-express` 7.9.1 is released :)